Lib/idlelib/config_key.py

```diff
@@ -186,7 +186,7 @@ def ClearKeySeq(self):
 
     def LoadFinalKeyList(self):
         #these tuples are also available for use in validity checks
-        self.functionKeys=('F1','F2','F2','F4','F5','F6','F7','F8','F9',
+        self.functionKeys=('F1','F2','F3','F4','F5','F6','F7','F8','F9',
```

Comment:
This should be in another issue (?), or do you want to PEP8 them, too? (I think this can be PEP8, too)